_data/harshmittal2210/harshmittal2210.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Harsh Mittal # No longer than 28 characters
+institution: The LNM Institute of Information Technology 🚩 # no longer than 58 characters
```

Comment:
Remove the 🚩(flag) as it is a non-ascii character.

Reply:
Sure, I'll just remove it. I took it from the template in Readme file.